core/app/models/spree/order.rb

```diff
@@ -400,7 +400,7 @@ def deliver_order_confirmation_email
 
     # Helper methods for checkout steps
     def paid?
-      payment_state == 'paid' || payment_state == 'credit_owed'
+      %w(paid credit_owed).include?(payment_state)
```

Comment:
My preference would be we let things settle for a few more days/weeks before we global change stuff. I reckon there are going to be a pretty large amount of people looking to cherry-pick changes in spree 3 or master and adding something white noisy to the merges isn't worth it being in a few days early. And then anything in those cherry picks would be globally fixed as well.